docs/manual/install/arch.rst

```diff
@@ -20,39 +20,33 @@ if you use `yaourt`_:
     # or for develop
     yaourt -S qtile-python3-git
 
-Using pacman
-============
+Using makepkg
+=============
 
-.. code-block:: bash
-
-    sudo pacman -S python pango python-cairocffi python-xcffib
-
-Also you need one qtile package from the AUR:
-
-- `qtile-python3-git`_ 
-- `qtile`_ 
+The latest version of either package can be obtained by downloading a snapshot
+or cloning its repository:
 
+.. code-block:: bash
 
-Installing AUR packages without helper
-======================================
+    # snapshot
+    curl -s https://aur.archlinux.org/cgit/aur.git/snapshot/<package-name>.tar.gz | tar -xvzf -
+    # or repository
+    git clone https://aur.archlinux.org/<package-name>.git
 
-To install these packages, download the .tar.gz's from the AUR and run the
-following commands for each:
+Next makepkg has to be called in the directory where the files were saved. It
+installs missing dependencies using pacman, builds the package, installs it
+and removes obsolete build-time dependencies afterwards:
 
 .. code-block:: bash
 
-    tar -xvzf <packagename>-<vernum>.tar.gz
-    cd <packagename>-<vernum>
-    makepkg -s
-    sudo pacman -U <packagename>
+    cd <package-name>
+    makepkg -sri
 
 Please see the Arch Wiki for more information on installing packages from
 the AUR:
-
-http://wiki.archlinux.org/index.php/AUR#Installing_packages
+https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages
```

Comment:
This could be turned into a pretty link like the others in the page, by using the word `AUR` before the colon as the link text.